added suggested replies doc (#1381)
airyhq/airy: docs/docs/ui/suggestedReplies.md

---
title: Suggested Replies
sidebar_label: Suggested Replies
---

import useBaseUrl from '@docusaurus/useBaseUrl';
import TLDR from "@site/src/components/TLDR";
import ButtonBox from "@site/src/components/ButtonBox";
import BoltSVG from "@site/static/icons/bolt.svg";
import RasaSVG from "@site/static/icons/rasa.svg";

<TLDR> Suggested Replies allow you to add predefined responses to conversation messages, thus improving the response time and the overall communication experience.</TLDR>

> Suggested Replies are text replies that are added to one of the last 5 messages in a conversation. They are added via API request and only support text content.

Reviewer: I think it supports any type of message

> Suggested Replies are text replies that are added to one of the last 5 messages in a conversation. They are added via API request and only support text content.

Reviewer 2: ```suggestion
Suggested Replies are text replies that you can add to any message via the HTTP API. The suggested replies of the last 5 messages of a given conversation are considered for suggestion.
```

> As soon as suggested replies have been successfully added to a message (i.e. the API request returned a 200 response), a Suggestions button appears in the conversation's UI, at the top left of the input, enabling the user to choose a suggested reply as answer.

Reviewer 2: ```suggestion
As soon as suggested replies have been successfully added to a message (i.e. the API request returned a 200 response), a suggestions button appears at the top left of the input in the messages list of conversation's UI, enabling the user to choose a suggested reply as answer.
```

The screenshot below shows a sample conversation in the UI of an Airy Core instance: two suggested replies have been added to the contact's last message ('Great that it worked. Is there anything else you need?' and 'Have a nice day!'). The user clicks on the Suggestions button at the top left of the conversation's input and selects the suggested reply 'Have a nice day!'.

<img alt="suggestedReplies" src={useBaseUrl('img/ui/suggested-replies.gif')}/>

## API request to add suggested replies

<ButtonBox
icon={<BoltSVG />}

> title='Channels endpoint'

Reviewer 2: ```suggestion
title='Suggest replies endpoint'
```
